extensions/amp-user-notification/amp-user-notification.md

```diff
@@ -121,11 +121,14 @@ will be passed in future requests to data-show-if-href)
 #### JSON Fields
 
 - `elementId` (string) - The HTML id used on `amp-user-notification` element.
```

Comment:
Small nit: I think the correct capitalization for id is ID. It looks super weird, but I still fairly sure it's the correct capitalization.